.gitlab-ci.yml

```diff
@@ -303,19 +303,22 @@ test:check-python3-formatting:
   rules:
     - if: '$TEST_MENDER_DIST_PACKAGES == "true"'
 
+# This template setups the Docker based environment to test the packages
+# from a specified Debian version
 .test:pkgs:
   stage: test
   image: ${CI_DEPENDENCY_PROXY_DIRECT_GROUP_IMAGE_PREFIX}/docker:dind
   tags:
     #- mender-qa-worker-generic
     - hetzner-amd-beefy
   needs:
-    # Note that we are only testing packages from Debian bullseye
-    - "build:pkgs:device-components: [crosscompile, debian, bullseye, amd64]"
-    - "build:pkgs:device-components: [crosscompile, debian, bullseye, armhf]"
-    - "build:pkgs:workstation-tools: [crosscompile, debian, bullseye, amd64]"
+    - "build:pkgs:device-components: [crosscompile, debian, amd64]"
+    - "build:pkgs:device-components: [crosscompile, debian, armhf]"
+    - "build:pkgs:workstation-tools: [crosscompile, debian, amd64]"
```

Comment:
This is technically wrong, it will never exist such a dependency `build:pkgs:device-components: [crosscompile, debian, amd64]`. It works because all final CI jobs do override `needs`.

Remove `needs` from `.test:pkgs:` hidden CI job (or set to emtpy `[]` as we do in other places, which is also useless but maybe more intuitive for newcomers - you tell me)